scss/civicrm/dashboard/pages/_edit-dashboard.scss

```diff
@@ -0,0 +1,92 @@
+#help {
```

Comment:
also, why is this `#help` styled again? This is a very generic component, why is it even styled here?